src/Pages/HomePage.tsx

```diff
@@ -1,12 +1,11 @@
 import { Box, Typography } from '@mui/material'
 import { useState } from 'react'
-import movieData from '../../data/movies.json'
+import { default as movieData } from '../../data/movies.json'
```

Comment:
I can't reach your branch for some reason, but do we need to change the name on movieData?

Reply:
I am not sure where this updated import of movieData is coming from... let's check with the other group members.